core-bundle/src/DependencyInjection/Configuration.php

```diff
@@ -82,6 +86,7 @@ static function (string $v): int {
                 ->end()
                 ->scalarNode('preview_script')
                     ->cannotBeEmpty()
+                    ->info('Entry point script that bypasses the front end cache for preview features, if necessary. The Contao Managed Edition uses "preview.php" here, but it can be customize for a regular Symfony application.')
```

Comment:
Should that be "_but it can be customize**d**_", right?

Reply:
Right ;)